### PR TITLE
enhancement: Genshin module description in readme

### LINKED ISSUE #51: [Enhancement]: Update README with Genshin module information
### My idea

Add information from https://github.com/Gwardia-Czapli/GwardiaHub/discussions/24#discussioncomment-9145332 to `README.md` (to `for genshin impact players` section).

### PR DESCRIPTION
**Pull request by Sanielia**
Based on https://github.com/Gwardia-Czapli/GwardiaHub/discussions/24 :33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ We want to create website with Django, that will be useful to as many as three d
   - place to submit feedback about meetings,
   - assignments given on meetings.
 - For Genshin Impact players:
-  - TBD
+  - wish simulator with character collection book,
+  - banners and events info,
+  - daily check in reminder,
+  - Genshin inspired this module exclusive UI.
```
Comment (Norbiros) [inline README.md]: Add `new uwu color theme`

File: README.md [hunk 2]
```diff
@@ -19,7 +19,10 @@ We want to create website with Django, that will be useful to as many as three d
   - place to submit feedback about meetings,
   - assignments given on meetings.
 - For Genshin Impact players:
-  - TBD
+  - wish simulator with character collection book,
```
Comment (Norbiros) [inline README.md]: Add more information for normal (not addicted to genshin) people. IDK what are those "wishes" :(

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Sanielia) [reply README.md]: wish is like lootbox, should I add this information to text??

Comment (goteusz-maszyk) [reply README.md]: yes

**Review by Norbiros — APPROVED** [review]

**Review by tapczanek1321 — APPROVED** [review]

**Pull request merged**